tests/Regression/GitHub/503.phpt

```diff
@@ -25,8 +25,8 @@ Failed asserting that two strings are identical.
 +++ Actual
 @@ @@
  #Warning: Strings contain different line endings!
- 'foo
- '
+-'foo
++'foo
```

Comment:
This change looks good to.
On `diff` v1.4 when using different line endings for a line the line was outputted with any of the two line breaks used.
On `diff` 2.x it lists both lines (with correct `-` or `+` prefix) with their linebreaks, making it possible to "see" the different line breaks.